include/clang/CrossTU/CrossTranslationUnit.h

```diff
@@ -156,7 +156,7 @@ class CrossTranslationUnitContext {
       ASTUnitImporterMap;
   CompilerInstance &CI;
   ASTContext &Context;
-  std::unique_ptr<ASTImporterLookupTable> LookupTable;
+  std::shared_ptr<ASTImporterSharedState> ImporterSharedSt;
```

Comment:
Probably name this `ImporterSharedState`, or have `lazyInitASTImporterSharedSt`.

Reply:
Ok, I renamed the function.